lib/cron.dart

```diff
@@ -120,8 +132,12 @@ class _Cron implements Cron {
     if (_closed) return;
     if (_timer != null || _schedules.isEmpty) return;
     final now = DateTime.now();
-    final ms = _millisecondsPerMinute -
-        (now.millisecondsSinceEpoch % _millisecondsPerMinute);
+    final isTickMinute = _schedules.any((task) =>
```

Comment:
Maybe I am wrong (would be nice to have tests, a big thing to fix), but this looks like wrong. If `task.schedule.seconds` is `null`, this expression will be false, which in the end will result `isTickMinute` being `false`. Could you please double-check?

Reply:
@isoos It's right that `isTickMinute` is `false` when `task.schedule.seconds` is `null`, because it means match all.
In class `Schedule`, if `cronFormat` is `* * * * * *` then `Schedule.seconds` is null.